code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -339,7 +343,8 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	icon_on = "pipeon"  //Note - these are in masks.dmi
 	icon_off = "pipeoff"
 	smoketime = 0
-	chem_volume = 50
+	chem_volume = 51
```

Comment:
50

Reply:
it'd be a good idea to comment why the values are 1 above what they should be

Reply:
To explain why, the author was concerned about the number of tasting nicotine messages being one off and added one to all them to fix it until I tell him that's not the solution and the code that emits these messages has thresholds which the last message will not go out due it. You can check it at code\modules\mob\living\carbon\taste.dm